From 418258a7dade570507dd4ebce46adc6c7310d546 Mon Sep 17 00:00:00 2001
From: Lei Maohui <leimaohui@fujitsu.com>
Date: Thu, 10 Jul 2025 06:37:21 +0000
Subject: [PATCH] Fix install error :

| sed: can't read build-ubinux-daily/tmp/work/ubinux_armv8-ubinux-linux/openvswitch/3.5.0/image/usr/lib64/openvswitch/ptest/tests/testsuite: No such file or directory

Upstream-Status: Pending
Signed-off-by: Liu Yiding <liuyd.fnst@fujitsu.com>
---
 test.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test.mk b/test.mk
index 8214219b6..3a8fd5782 100644
--- a/test.mk
+++ b/test.mk
@@ -39,7 +39,7 @@ test-install: $(TEST_DEPEND)
 	install $(srcdir)/utilities/ovs-pki.in $(TEST_DEST)/utilities ;\
 	for p in $$list ; do \
 	  echo $$p ;\
-	  p=$${p#../git/} ;\
+	  p=$${p#../sources/openvswitch*/} ;\
 	  pre=$${p#tests\/} ;\
 	  if test $$pre != $$p ; then \
 	    echo installing $$p to $(TEST_DEST)/tests/$$pre ;\
-- 
2.43.0

